app/ide-desktop/lib/content/src/index.html

```diff
@@ -27,12 +27,25 @@
             user-scalable = no"
     />
     <title>Enso</title>
+    <!--
+      TODO [NP]: https://github.com/enso-org/cloud-v2/issues/344
+      Ask @mwu how to get TailwindCSS to play nice with the fact that `lib/authentication` is a
+      dependency of `lib/content` and `lib/cloud` and not its own standalone package, so that we can
+      use `@tailwind` directives in the `index.html` file.
+    -->
+    <script src="https://cdn.tailwindcss.com"></script>
     <link rel="stylesheet" href="/style.css" />
     <link rel="stylesheet" href="/docsStyle.css" />
     <script type="module" src="/index.js" defer></script>
     <script type="module" src="/run.js" defer></script>
   </head>
   <body>
+    <!--
+      This `div` is used to render the authentication/dashboard app. It is kept separate from the
+      `root` `div`, which used used to render the IDE. This is done to ensure the two do not
```

Comment:
1. There are so many English mistakes in all the docs!
2. `This is done to ensure the two do not interfere with each other.` - please review ALL docs in this PR, because I see it in many places and we will probably have problem with it during next re-reviews. Make sure all sentences tell "things". Like, this sentence does not tell anything ,as the word "interfere" is not defined and it is not iunderstandable what you mean here.